entries/checked-selector.xml

```diff
@@ -7,7 +7,8 @@
   </signature>
   <desc>Matches all elements that are checked or selected.</desc>
   <longdesc>
-    <p>The <code>:checked</code> selector works for checkboxes, radio buttons, and select elements. For select elements only, use the <code>:selected</code> selector.</p>
+    <p>The <code>:checked</code> selector works for checkboxes, radio buttons, and "selected" options of select elements.</p>
```

Comment:
The original description is incorrect in that it claims `:selected` will return the `select` element that is "selected". The nuance is that the "selected" `option` element(s) is/are returned.

I'm using double quotes in an attempt to differentiate between the three versions of select:
`:select` --the selector
`select` --the element
`selected` --the `<option>` attribute